src/App.tsx

```diff
@@ -1,8 +1,112 @@
-import React from 'react';
+import React, { useState, FC } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { getData } from './api/api';
 import './App.css';
+import { Todo } from './interfaces/TodoInterface';
+import { User } from './interfaces/UserInterface';
+import { TodoList } from './components/TodoList/TodoList';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setLoading] = useState<boolean>(false);
+  const [completedCounter, setCompletedCounter] = useState(0);
+  const [titleCounter, setTitleCounter] = useState(0);
+  const [userCounter, setUserCounter] = useState(0);
+
+  const loadData = async () => {
+    setLoading(true);
+
+    const allTodos = await getData<Todo>('todos');
+    const allUsers = await getData<User>('users');
```

Comment:
like for generic types
```suggestion
    const fetchedTodos = await getData<Todo>('todos');
    const fetchedUsers = await getData<User>('users');
```